### PR TITLE
Add fix to ensure token manager hook is set before allowing staking.

### PR DESCRIPTION
**Pull request by willjgriff**
Fix for recent vulnerability. Including test.

### CODE REVIEW

File: package.json
Comment (fabriziovigevani) [inline package.json:6]: Is there a branch with this new version? I guess this new version only exposes the `getTokenManager` function?

**Review by fabriziovigevani — APPROVED** [review]
LGTM!

**Review by rperez89 — APPROVED** [review]
Good improvement! lgtm

File: package.json
Comment (willjgriff) [reply package.json:6]: Yeah exactly, it is on this branch: https://github.com/1Hive/token-manager-app/tree/latest-aragon-os which I was waiting until the aragon OS is out of release candidate stage until I merged. Although I probably could just merge it, just concerned it might mess up anyone looking at it when using the gardens template or old conviction voting which uses the old aragon OS is all.

**Pull request merged**
